TextIOWrapper.encoding missing Optional Argument

Happy to submit a PR to fix but I think this is wrong

```python
from io import BytesIO, TextIOWrapper
from typing import Optional

encoding = None  # type: Optional[str]
buf = BytesIO()
TextIOWrapper(buf, encoding=encoding)
```

```sh
mypy_test.py:6: error: Argument "encoding" to "TextIOWrapper" has incompatible type "Optional[str]"; expected "str"
```

--- a/stdlib/3/io.pyi
+++ b/stdlib/3/io.pyi
@@ -159,7 +159,7 @@ class TextIOWrapper(TextIO):
     def __init__(
         self,
         buffer: IO[bytes],
-        encoding: str = ...,
+        encoding: Optional[str] = ...,
         errors: Optional[str] = ...,
         newline: Optional[str] = ...,
         line_buffering: bool = ...,

